File: 2021년/6월/0630 정수 제곱근 판별.py
# 프로그래머스_정수 제곱근 판별_수학_레벨 1

import math


def solution(n):
    num = math.sqrt(n)
    if num == int(num):
        return (num+1)**2
    else:
        return -1


print(solution(3))

File: 2021년/6월/0630 정수 제곱근 판별.cpp
// 프로그래머스_정수 제곱근 판별_수학_레벨 1

#include<cmath>

long long solution(long long n) {
    
    double num = sqrt(n);
    
    if (num == int(num)) return (num + 1) * (num + 1);
    else return -1;

}
